addon-test-support/@ember/test-helpers/dom/focus.js

```diff
@@ -0,0 +1,41 @@
+import getElement from './-get-element';
+import fireEvent from './fire-event';
+import settled from '../settled';
+import isFocusable from './-is-focusable';
+
+/*
+  @method focus
+  @param {String|HTMLElement} selector
+  @return {RSVP.Promise}
+  @public
+*/
+export default function focus(selectorOrElement) {
+  if (!selectorOrElement) {
+    throw new Error('Must pass an element or selector to `focus`.');
+  }
+
+  let element = getElement(selectorOrElement);
+
+  if (isFocusable(element)) {
```

Comment:
should this throw an error or warning if the element is not focusable?

Reply:
Absolutely